### ISSUE TITLE
Error on resizing

### ISSUE THREAD

**Issue by GerritK** [BTMorton/angular2-grid#242]
After resizing a grid-item i got this error:

```
ERROR TypeError: this._itemGrid[y] is undefined
Stack-Trace:
./node_modules/angular2-grid/directives/NgGrid.js/NgGrid.prototype._removeFromGrid@http://localhost:4200/vendor.bundle.js:61187:17
./node_modules/angular2-grid/directives/NgGrid.js/NgGrid.prototype.updateItem@http://localhost:4200/vendor.bundle.js:60575:9
./node_modules/angular2-grid/directives/NgGridItem.js/NgGridItem.prototype.setConfig@http://localhost:4200/vendor.bundle.js:61646:13
./node_modules/angular2-grid/directives/NgGridItem.js/NgGridItem.prototype._applyChanges@http://localhost:4200/vendor.bundle.js:61827:9
./node_modules/angular2-grid/directives/NgGridItem.js/NgGridItem.prototype.ngDoCheck@http://localhost:4200/vendor.bundle.js:61656:17
```

**Comment by BTMorton** [comment]
Issue should be resolved in v2.0.4

**Comment by GerritK** [comment]
The error is fixed but now i can't get bigger than the minimum size of the item.
